website/docs/releases/migration-4.md

```diff
@@ -0,0 +1,69 @@
+# Migration guide from 3.x to 4.x
+
+
+## Backward incompatible changes
+
+Minimal required versions are:
+
+- Typescript (if used on the project): 4.1
+
+### Extractor configuration changes:
+
+The big change in the v4 is changes in extractor internals. Now it less fragile, and not depends on the host project settings.
+
+For most projects it should work without extra configuration as long as it valid ES code.
+
+`extractorBabelOptions` is not usefull anymore, please delete it from your config.
+
+
+```diff title="lingui.config.js"
+module.exports = {
+  - extractorBabelOptions: { [...] }
```

Comment:
the diffs are not highlighted, please check the preview - https://js-lingui-git-fork-thekip-refactor-extractor-babel-lingui.vercel.app/releases/migration-4

Reply:
done